kits/live-api-debugger/README.md

```
@@ -0,0 +1,18 @@
+# Live API Debugger 🐛 -> 🦋
+
+An advanced developer tool that fixes your broken API integration code by directly reading the live official API documentation.
+
+## Why this exists
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)